apps/www/registry/default/ui/input-otp.tsx

```diff
@@ -0,0 +1,64 @@
+"use client"
+
+import * as React from "react"
+import { OTPInput, SlotProps } from "input-otp"
+import { Dot } from "lucide-react"
+
+import { cn } from "@/lib/utils"
+
+const InputOTP = React.forwardRef<
+  React.ElementRef<typeof OTPInput>,
+  React.ComponentPropsWithoutRef<typeof OTPInput>
+>(({ className, ...props }, ref) => (
+  <OTPInput
+    ref={ref}
+    containerClassName={cn("flex items-center gap-2", className)}
```

Comment:
Add `has-[:disabled]:opacity-30` for disabled modifier appearance.
PS: do the same for `registry/default` and `registry/new-york`

@shadcn 

```diff
-containerClassName={cn("flex items-center gap-2", className)}
+containerClassName={cn("flex items-center gap-2 has-[:disabled]:opacity-30", className)}
```

Reply:
@shadcn I have added the modifier for you in this PR. https://github.com/shadcn-ui/ui/pull/2938